specs/packagemanager/PackageManagement.md

```diff
@@ -344,6 +347,12 @@ package management APIs in Windows (e.g. Windows.Management.Deployment.PackageMa
   not PackageFamilyName. `PackageDeploymentManager` provides a richer API accepting additional
   identifiers.
 
+## 3.12 Is*Provisioned()
+
+Is\*Provisioned\*() methods determine if the target is provisioned.
+
+These methods require administrative privileges.
```

Comment:
If I pass a package that is not even downloaded (much less provisioned), does it fail with "Invalid package name" or does it return false?

Reply:
False

Whether absent or present but not provisioned it's still "not provisioned". If you asked is Raymond in the kitchen it doesn't matter if Raymon is in another room in the house or not in the house at all, either way he's not in the kitchen :-)

If you pass in invalid data (e.g. something that's not a syntactically valid pkgfamilyname, pkgfullname or URI) you get E_INVALIDARG.